api/src/main/java/io/smallrye/stork/api/Stork.java

```diff
@@ -0,0 +1,47 @@
+package io.smallrye.stork.api;
+
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * The central API of Stork
+ */
+public interface Stork {
```

Comment:
Won't having interface named the same as the central entrypoint class reduce discoverability?

Reply:
Maybe the interface could be called `StorkServiceRegistry`?

Reply:
I agree the name is not great, and it's confusing.

The problem is that we are leaking the Stork from the core API (because of the `initialize` method) and we also have an API module, which does not contain the main API... 

`StorkServiceRegistry` would work as name. I will rename it.